metadata-ingestion/src/datahub/ingestion/source/dbt/dbt_cloud.py

```diff
@@ -56,12 +79,18 @@ class DBTCloudConfig(DBTCommonConfig):
         description="The dbt Cloud project ID to use.",
     )
 
-    job_id: int = Field(
-        description="The ID of the job to ingest metadata from.",
+    job_id: Optional[int] = Field(
+        None,
+        description="The ID of the job to ingest metadata from. Required in explicit mode (when auto_discovery is disabled).",
```

Comment:
wondering if we should just deprecate this

auto discovery mode + job_id_pattern also covers the case of ingesting single job id

we miss run_id though